### PR TITLE
SEOImages: User Story 1463880, Part 29

### PR DESCRIPTION
**Pull request by damabe**
## Summary

Updating image file names and references to improve SEO.

Fixes #1463880

Contributes to #10839

### CODE REVIEW

**Comment by damabe** [comment]
@mairaw This PR is ready for review. CATS report: https://contentqacats.azurewebsites.net/CheckResults/2019032000036/Report/

**Comment by damabe** [comment]
Updated based on other PR reviews. CATS report: https://contentqacats.azurewebsites.net/CheckResults/2019032700051/Report/

File: docs/framework/resources/creating-satellite-assemblies-for-desktop-apps.md [hunk 1]
```diff
@@ -46,10 +46,11 @@ Resource files play a central role in localized applications. They enable an app
...
+ The following illustration shows a sample directory structure and location requirements for applications that you are not installing in the [global assembly cache](../../../docs/framework/app-domains/gac.md). The items with .txt and .resources extensions will not ship with the final application. These are the intermediate resource files used to create the final satellite resource assemblies. In this example, you could substitute .resx files for the .txt files. For more information, see [Packaging and Deploying Resources](../../../docs/framework/resources/packaging-and-deploying-resources-in-desktop-apps.md). 
+ 
+ The following image shows the satellite assembly directory:
   
- ![Satellite assemblies](../../../docs/framework/resources/media/satelliteassemblydir.gif "satelliteassemblydir")  
-Satellite assembly directory  
+ ![Diagram that shows a satellite assembly directory.](./media/creating-satellite-assemblies-for-desktop-apps/satellite-assembly-directory.gif
```
Comment (mairaw) [inline docs/framework/resources/creating-satellite-assemblies-for-desktop-apps.md]:
```suggestion
 ![A satellite assembly directory with localized cultures subdirectories.](./media/creating-satellite-assemblies-for-desktop-apps/satellite-assembly-directory.gif)
```

File: docs/framework/winforms/controls/datagrid-control-overview-windows-forms.md [hunk 2]
```diff
@@ -29,10 +29,9 @@ ms.assetid: 85604bce-bc03-49d9-9030-dda8896c44b1
   
  If the <xref:System.Windows.Forms.DataGrid> is bound to data with multiple related tables, and if navigation is enabled on the grid, the grid will display expanders in each row. With an expander, the user can move from a parent table to a child table. Clicking a node displays the child table, and clicking a back button displays the original parent table. In this manner, the grid displays the hierarchical relationships between tables.  
   
- The following screen shot shows a DataGrid bound to data with multiple tables.  
+ The following screen shot shows a DataGrid bound to data with multiple tables:
```
Comment (mairaw) [inline docs/framework/winforms/controls/datagrid-control-overview-windows-forms.md]:
```suggestion
 The following screenshot shows a DataGrid bound to data with multiple tables:  
```

File: docs/framework/winforms/controls/datagrid-control-overview-windows-forms.md [hunk 3]
```diff
@@ -29,10 +29,9 @@ ms.assetid: 85604bce-bc03-49d9-9030-dda8896c44b1
   
  If the <xref:System.Windows.Forms.DataGrid> is bound to data with multiple related tables, and if navigation is enabled on the grid, the grid will display expanders in each row. With an expander, the user can move from a parent table to a child table. Clicking a node displays the child table, and clicking a back button displays the original parent table. In this manner, the grid displays the hierarchical relationships between tables.  
   
- The following screen shot shows a DataGrid bound to data with multiple tables.  
+ The following screen shot shows a DataGrid bound to data with multiple tables:  
   
- ![A DataGrid bound to data with multiple tables](./media/vbcontrol1.gif "vbControl1")  
-A DataGrid bound to data with multiple tables  
+ ![Diagram that shows a DataGrid bound to data with multiple tables.](./media/datagrid-control-overview-windows-forms/datagrid-bound-multiple-tables.gif)
```
Comment (mairaw) [inline docs/framework/winforms/controls/datagrid-control-overview-windows-forms.md]:
```suggestion
 ![A WinForms app showing a DataGrid bound to data with multiple tables.](./media/datagrid-control-overview-windows-forms/datagrid-bound-multiple-tables.gif)  
```

File: docs/visual-basic/programming-guide/language-features/control-flow/decision-structures.md [hunk 4]
```diff
@@ -15,8 +15,7 @@ Visual Basic lets you test conditions and perform different operations depending
   
  The following illustration shows a decision structure that tests for a condition being true and takes different actions depending on whether it is true or false.  
   
- ![Flow chart of an If...Then...Else construction](../../../../visual-basic/programming-guide/language-features/control-flow/media/ifthenelse.gif "IfThenElse")  
-Taking different actions when a condition is true and when it is false  
+ ![Diagram that shows an If...Then...Else construction.](./media/decision-structures/if-then-else-construction.gif)
```
Comment (mairaw) [inline docs/visual-basic/programming-guide/language-features/control-flow/decision-structures.md]:
```suggestion
 ![A flow chart of an If...Then...Else construction.](./media/decision-structures/if-then-else-construction.gif)  
```

File: docs/visual-basic/programming-guide/program-structure/comments-in-code.md [hunk 5]
```diff
@@ -48,7 +48,7 @@ As you read the code examples, you often encounter the comment symbol (`'`). Thi
   
 -   Comments cannot follow a line-continuation sequence on the same line.  
   
- You can add or remove comment symbols for a block of code by selecting one or more lines of code and choosing the **Comment** (![VisualBasicWinAppCodeEditorCommentButton](../../../visual-basic/programming-guide/program-structure/media/vacommentbutton.gif "vaCommentButton")) and **Uncomment** (![VisualStudioWinAppProjectUncommentButton](../../../visual-basic/programming-guide/program-structure/media/vauncommentbutton.gif "vaUncommentButton")) buttons on the **Edit** toolbar.  
+ You can add or remove comment symbols for a block of code by selecting one or more lines of code and choosing the **Comment** (![Graphic shows the Visual Basic Comment button.](./media/comments-in-code/visual-basic-comment-button.gif)) and **Uncomment** (![Graphic shows the Visual Basic Uncomment button.](./media/comments-in-code/visual-basic-uncomment-button.gif)) buttons on the **Edit** toolbar.
```
Comment (mairaw) [inline docs/visual-basic/programming-guide/program-structure/comments-in-code.md]:
```suggestion
 You can add or remove comment symbols for a block of code by selecting one or more lines of code and choosing the **Comment** (![The Visual Basic Comment button in Visual Studio.](./media/comments-in-code/visual-basic-comment-button.gif)) and **Uncomment** (![The Visual Basic Uncomment button in Visual Studio.](./media/comments-in-code/visual-basic-uncomment-button.gif)) buttons on the **Edit** toolbar.  
```

**Review by mairaw — CHANGES_REQUESTED** [review]
Thanks @damabe. A few comments before merging this one.

**Comment by damabe** [comment]
@mairaw I've made these changes. New CATS: https://contentqacats.azurewebsites.net/CheckResults/2019040400032/Report/

**Review by mairaw — APPROVED** [review]
Thanks @damabe!

**Pull request merged**